Gazebo.md

```diff
@@ -338,242 +338,242 @@
 [ign-transport11-focal-actions-badge]: https://github.com/gazebosim/gz-transport/workflows/Ubuntu%20CI/badge.svg?branch=ign-transport11&event=push
 [ign-utils1-repo]: https://github.com/gazebosim/gz-utils/tree/ign-utils1
-[ign-utils1-focal]: https://build.osrfoundation.org/job/ignition_utils-ci-ign-utils1-focal-amd64
-[ign-utils1-focal-badge]: https://build.osrfoundation.org/buildStatus/icon?job=ignition_utils-ci-ign-utils1-focal-amd64
+[ign-utils1-focal]: https://build.osrfoundation.org/job/gz_utils-ci-ign-utils1-focal-amd64
+[ign-utils1-focal-badge]: https://build.osrfoundation.org/buildStatus/icon?job=gz_utils-ci-ign-utils1-focal-amd64
 [ign-utils1-windows7]: https://build.osrfoundation.org/job/ign_utils-ign-1-win
 [ign-utils1-windows7-badge]: https://build.osrfoundation.org/buildStatus/icon?job=ign_utils-ign-1-win
 [ign-utils1-homebrew]: https://build.osrfoundation.org/job/ignition_utils-ci-ign-utils1-homebrew-amd64
 [ign-utils1-homebrew-badge]: https://build.osrfoundation.org/buildStatus/icon?job=ignition_utils-ci-ign-utils1-homebrew-amd64
 [ign-utils1-focal-actions]: https://github.com/gazebosim/gz-utils/actions?query=workflow%3A%22Ubuntu+CI%22+branch%3Aign-utils1+event%3Apush
 [ign-utils1-focal-actions-badge]: https://github.com/gazebosim/gz-utils/workflows/Ubuntu%20CI/badge.svg?branch=ign-utils1&event=push
 [sdformat12-repo]: https://github.com/gazebosim/sdformat/tree/sdf12
-[sdformat12-focal]: https://build.osrfoundation.org/job/sdformat-ci-sdformat12-focal-amd64
-[sdformat12-focal-badge]: https://build.osrfoundation.org/buildStatus/icon?job=sdformat-ci-sdformat12-focal-amd64
+[sdformat12-focal]: https://build.osrfoundation.org/job/sdformat-ci-sdf12-focal-amd64
+[sdformat12-focal-badge]: https://build.osrfoundation.org/buildStatus/icon?job=sdformat-ci-sdf12-focal-amd64
 [sdformat12-windows7]: https://build.osrfoundation.org/job/sdformat-sdf-12-win
 [sdformat12-windows7-badge]: https://build.osrfoundation.org/buildStatus/icon?job=sdformat-sdf-12-win
 [sdformat12-homebrew]: https://build.osrfoundation.org/job/sdformat-ci-sdformat12-homebrew-amd64
 [sdformat12-homebrew-badge]: https://build.osrfoundation.org/buildStatus/icon?job=sdformat-ci-sdformat12-homebrew-amd64
 [sdformat12-focal-actions]: https://github.com/osrf/sdformat/actions?query=workflow%3A%22Ubuntu%22+branch%3Asdf12+event%3Apush
 [sdformat12-focal-actions-badge]: https://github.com/osrf/sdformat/workflows/Ubuntu/badge.svg?branch=sdf12&event=push
 [ign-common3-repo]: https://github.com/gazebosim/gz-common/tree/ign-common3
-[ign-common3-focal]: https://build.osrfoundation.org/job/ignition_common-ci-ign-common3-focal-amd64
-[ign-common3-focal-badge]: https://build.osrfoundation.org/buildStatus/icon?job=ignition_common-ci-ign-common3-focal-amd64
+[ign-common3-bionic]: https://build.osrfoundation.org/job/ign_common-ci-ign-common3-bionic-amd64
+[ign-common3-bionic-badge]: https://build.osrfoundation.org/buildStatus/icon?job=ign_common-ci-ign-common3-bionic-amd64
```

Comment:
Those names have a reference to ign, is that expected?

Reply:
Same for all below

Reply:
I still see references to `ign` namespace on the job list for old gazebo distributions. I think that given that that is the current state of the buildfarm this would be okay to merge. For Harmonic I don't think we have moved all of the jobs to the `gz` namespace either there are references to `ign` on windows and some homebrew jobs.